sammba/registration/struct.py

```diff
@@ -320,8 +320,13 @@ def anats_to_common(anat_filenames, write_dir, brain_volume,
     out_undump = undump(in_file=out_tstat.outputs.out_file,
                         out_file=os.path.join(write_dir, 'undump.nii.gz'),
                         outputtype='NIFTI_GZ')
-    out_refit = refit(in_file=out_undump.outputs.out_file,
-                      xorigin='cen', yorigin='cen', zorigin='cen')
+    import subprocess
```

Comment:
I fixed this issue in PR #89. Please do not worry about this bug as this code will be rebased at some point on the fix